TableBookingAPI/TableBooking.Model/Dtos/BookingDtos/BookingDto.cs

```diff
@@ -7,7 +7,7 @@ public class BookingDto
         public Guid Id { get; set; }
         public DateTime Date { get; set; }
         public int DurationInMinutes { get; set; }
-        public TableDto TableDto { get; set; }
```

Comment:
[PL] Model Tabel dla rezerwacji jest (na razie) bezużyteczny, ponieważ Table jest bardziej skupiona na modelu Restaurant. Użytkownik posiada Booing, która nie musi posiadać modelu całego Table. Zachowajmy prostotę.

[ENG] Table model for Booking is useless (for now), because Table is more focused on Restaurant model. User has a Booking which doesn't have to have whole Table model. Keep things simple.